packages/vue/src/components/range/DynamicRangeSlider.jsx

```diff
@@ -1,12 +1,11 @@
 import VueTypes from 'vue-types';
+import VueSlider from 'vue-slider-component';
 import { Actions, helper } from '@appbaseio/reactivecore';
-import NoSSR from 'vue-no-ssr';
 import Container from '../../styles/Container';
```

Comment:
You removed `vue-no-ssr`, Please make sure it works well with the server-side rendering.
Just test the `with-nuxt` example.
If it works then remove this package from `RangeSlider` component & `package.json`.